### DISCUSSION TITLE
tiling and window mode ?

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3395] [Ideas]
is it possible ? window mode and tiling mode ??
![2022-03-01-015611_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/156072301-2df26267-3a0f-4c7d-85d2-27e05d02a652.png)

**Comment by m-col** [comment]
Hi again @ArchKubi! Could you explain a bit more about what you mean? It is unclear

**Reply by ghost** [reply]
there no window scale just tiling scale and ı'm poor ı use 1280x1012
